packages/cli/src/lib/quire/index.js

```diff
@@ -6,6 +6,7 @@ import { isEmpty } from '#helpers/is-empty.js'
 import fs from 'fs-extra'
 import git from '#src/lib/git/index.js'
 import inv from 'install-npm-version'
+import packageConfig from '../../../package.json' assert { type: 'json' }
```

Comment:
Can we use the npm subpath here?
```suggestion
import packageConfig from '#root/package.json' assert { type: 'json' }
```